cli.js

```diff
@@ -88,7 +89,8 @@ var cli = meow([
 		'require',
 		'timeout',
 		'source',
-		'match'
+		'match',
+		'pool-size'
```

Comment:
I think `concurrency` would be a better flag name than `pool-size`. It's the most common name for it that I've seen. For example: http://bluebirdjs.com/docs/api/promise.map.html#map-option-concurrency